src/compiler/types.ts

```diff
@@ -4331,6 +4331,8 @@ namespace ts {
         types?: string[];
         /** Paths used to compute primary types search locations */
         typeRoots?: string[];
+        /** Scopes used to search for type-only packages */
+        searchScopes?: string[];

```

Comment:
I think we need to talk about where this option lives, in the tsconfig or a sever commandline option that users specify in IDE configuration for instance..

if it is in tsconfig.json then it needs to be added to the commandline-parser to validate it..

if it a server option, then it needs to be in the server.ts instead.

Reply:
Agreed, but if it is not in the `tsconfig.json` then how will the compiler when invoked from the command line pull in typings from `node_modules/@companyTypes/companyPackageName`?

Reply:
is this for the commandline for for ATA (i.e. editor only scenarios)? may be you can explain the request one more time in more details.

Reply:
Both. I'd like to implement both pieces.